tests/mocha/event_procedure_parameter_delete_test.js

```diff
@@ -117,7 +116,7 @@ suite.skip('Procedure Parameter Delete Event', function() {
         const actualProc = this.createProcedureModel('test name', 'test id');
         this.procedureMap.add(actualProc);
 
-        event.run(true /* forward */);
+        event.run(false /* backward */);
```

Comment:
While I don't see any rule barring what you've done here, the [styleguide seems to recommend](https://google.github.io/styleguide/jsguide.html#formatting-param-name-comments):
```suggestion
        event.run(/* forward= */ false);
```
(here and below)